client/hwlib/src/models/request_validators.rs

```diff
@@ -255,7 +255,7 @@ mod tests {
             .collect::<String>();
 
         let mock_calls = vec![
-            ((constants::DPKG, vec!["--print-architecture"]), Ok("amd64")),
+            ((constants::ARCH, vec![]), Ok("x86_64")),
```

Comment:
the version has to be `amd64` as all our checkbox submissions use it

Reply:
In this case the test should call the conversion function because the final result has to be “amd64”